dev/PowerNotifications/PowerNotifications.idl

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// REVIEW: We're not doing doing any versioning within IDL/WinMDs, right?
```

Comment:
Please use //TODO:123 title of the Issue (or bug/task/deliverable if and ADO item)

Though in this case I suggest deleting the comment and following up offline

(same comment applies throughout)